de_web_plugin.cpp

```diff
@@ -6128,6 +6128,15 @@ void DeRestPluginPrivate::updateSensorNode(const deCONZ::NodeEvent &event)
 
                             if (i->modelId().startsWith(QLatin1String("lumi.vibration")) && i->type() == QLatin1String("ZHAVibration"))
                             {
+
+                                //Set tiltangle to 0 by defaut, but without queueEvent
```

Comment:
Not good: you need to send an event on every change, or rules won't trigger and clients that don't poll won't see the change.